meshnets/utils/model_training.py

```diff
@@ -78,28 +78,25 @@ def train_model(config, experiment_config, train_dataset, validation_dataset):
                                    persistent_workers=True,
                                    shuffle=False)
 
-    # Define the model
-    model = MeshGraphNet(
+    # Define the wrapper and the model
+    lightning_wrapper = MGNLightningWrapper(
```

Comment:
try just importing meshnets and expand the the path here to include all the packages.
This allow us to detect redundancies in the name of the pacakages and in the name of the classes.